ImperatorToCK3/Source/Mappers/ReligionMapper/ReligionMapping.h

```diff
@@ -13,10 +13,12 @@ class ReligionMapping: commonItems::parser
 
 	[[nodiscard]] const auto& getImperatorReligions() const { return impReligions; }
 	[[nodiscard]] const auto& getCK3Religion() const { return ck3Religion; }
+	[[nodiscard]] const auto& getCK3Regions() const { return ck3Regions; }
```

Comment:
Apparently I started work on adding region support to ReligionMapper, it's not finished though.